src/components/signup/components/Signup.js

```diff
@@ -61,6 +65,16 @@ class Signup extends React.Component {
                     onCancel={this.handleCancel}
                     onCreate={this.handleCreate}
                 />
+
+                {
+                    signupStatus !== undefined &&
+                    <span>
+                        {
+                            signupStatus ? message.success(signupMessage, 5) :
+                                message.error(signupMessage, 5)
+                        }
+                    </span>
+                }
```

Comment:
Oh! I was looking for where this logic went to from home.js

Reply:
I moved them to there individual components instead of having it at home